orchestrator/test_runner/src/main.rs

```diff
@@ -74,6 +79,14 @@ lazy_static! {
         env::var("COSMOS_NODE_GRPC").unwrap_or_else(|_| "http://localhost:9090".to_owned());
     static ref COSMOS_NODE_ABCI: String =
         env::var("COSMOS_NODE_ABCI").unwrap_or_else(|_| "http://localhost:26657".to_owned());
+    static ref IBC_ADDRESS_PREFIX: String =
```

Comment:
this block needs a comment stating what IBC means in this context (that we are running another chain in the test env)